magento/pwa-studio
# My Account Sign In.

## Description

Added singin section to the new My account trigger in the header section.

## Related Issue
Closes [PWA-625](https://jira.corp.magento.com/browse/PWA-625)

### Verification Stakeholders
@soumya-ashok 
@schensley 
@jimbo 

### Verification Steps
1. Go to the deployed app and click on the `Sign In` trigger in the header.
2. Sign in should work as expected.

## Screenshots / Screen Captures (if appropriate)

![image](https://user-images.githubusercontent.com/35203638/88968790-7c802180-d275-11ea-8f71-87cbff7daf9c.png)

![image](https://user-images.githubusercontent.com/35203638/88968841-928de200-d275-11ea-8f72-8cae318b271d.png)

![image](https://user-images.githubusercontent.com/35203638/89349811-75da1b80-d674-11ea-8ed4-25ac97ef5084.png)

![image](https://user-images.githubusercontent.com/35203638/89349797-6d81e080-d674-11ea-81c3-9d90447cdc3a.png)

## Checklist

* I have updated the documentation accordingly, if necessary.
